### DISCUSSION TITLE
Adding BeanBag to Explorer Keep getting declined for no testing but I did test on your environment

### DISCUSSION THREAD

**Discussion by briannichols0702** [WalletConnect/walletconnect-monorepo#3557] [Developer Support]
https://rpc.ankr.com/bsc\",\"100\":\"https://rpc.gnosischain.com\",\"137\":\"https://polygon-rpc.com\",\"250\":\"https://rpc.ankr.com/fantom\",\"324\":\"https://mainnet.era.zksync.io\",\"1088\":\"https://andromeda.metis.io/?owner=1088\",\"4689\":\"https://babel-api.mainnet.iotex.io\",\"9001\":\"https://eth.bd.evmos.org:8545\",\"42161\":\"https://arb1.arbitrum.io/rpc\",\"43114\":\"https://api.avax.network/ext/bc/C/rpc\"}}},\"relays\":[{\"protocol\":\"irn\"}],\"proposer\":{\"publicKey\":\"7ab9bf8837d6f33e1a447b56757cedce6f0f95a680d691b20f00e25581b8d533\",\"metadata\":{\"description\":\"\",\"url\":\"https://lab.web3modal.com\",\"icons\":[],\"name\":\"\"}}}},\"chainId\":null,\"expiry\":1695403692},{\"id\":1692811701684381,\"topic\":\"7204f0a50fc0229caf5886b9fa1add18fe4cdeebc130427d33ac5d356278f933\",\"request\":{\"method\":\"wc_sessionPropose\",\"params\":{\"requiredNamespaces\":{\"eip155\":{\"chains\":[\"eip155:1\"],\"methods\":[\"eth_sendTransaction\",\"personal_sign\"],\"events\":[\"chainChanged\",\"accountsChanged\"],\"rpcMap\":{\"1\":\"https://cloudflare-eth.com\"}}},\"optionalNamespaces\":{\"eip155\":{\"chains\":[\"eip155:43114\",\"eip155:100\",\"eip155:42161\",\"eip155:137\",\"eip155:56\",\"eip155:250\",\"eip155:324\",\"eip155:10\",\"eip155:9001\",\"eip155:4689\",\"eip155:1088\"],\"methods\":[\"eth_sendTransaction\",\"personal_sign\",\"eth_accounts\",\"eth_requestAccounts\",\"eth_sendRawTransaction\",\"eth_sign\",\"eth_signTransaction\",\"eth_signTypedData\",\"eth_signTypedData_v3\",\"eth_signTypedData_v4\",\"wallet_switchEthereumChain\",\"wallet_addEthereumChain\",\"wallet_getPermissions\",\"wallet_requestPermissions\",\"wallet_registerOnboarding\",\"wallet_watchAsset\",\"wallet_scanQRCode\"],\"events\":[\"chainChanged\",\"accountsChanged\",\"message\",\"disconnect\",\"connect\"],\"rpcMap\":{\"1\":\"https://cloudflare-eth.com\",\"10\":\"https://mainnet.optimism.io\",\"56\":\"https://rpc.ankr.com/bsc\",\"100\":\"https://rpc.gnosischain.com\",\"137\":\"https://polygon-rpc.com\",\"250\":\"https://rpc.ankr.com/fantom\",\"324\":\"https://mainnet.era.zksync.io\",\"1088\":\"https://andromeda.metis.io/?owner=1088\",\"4689\":\"https://babel-api.mainnet.iotex.io\",\"9001\":\"https://eth.bd.evmos.org:8545\",\"42161\":\"https://arb1.arbitrum.io/rpc\",\"43114\":\"https://api.avax.network/ext/bc/C/rpc\"}}},\"relays\":[{\"protocol\":\"irn\"}],\"proposer\":{\"publicKey\":\"ac52dfbc215e73f2cdfb15384cb98135a01c839afbe3fce6d9475d3831d6c73b\",\"metadata\":{\"description\":\"\",\"url\":\"https://lab.web3modal.com\",\"icons\":[],\"name\":\"\"}}}},\"chainId\":null,\"expiry\":1695403701},{\"id\":1692811753684700,\"topic\":\"2e7ee1019facdf743938936cb8f3c951168b5cb7266fcf16f67a4c74a3e4eda6\",\"request\":{\"method\":\"wc_sessionPropose\",\"params\":{\"requiredNamespaces\":{\"eip155\":{\"chains\":[\"eip155:1\"],\"methods\":[\"eth_sendTransaction\",\"personal_sign\"],\"events\":[\"chainChanged\",\"accountsChanged\"],\"rpcMap\":{\"1\":\"https://cloudflare-eth.com\"}}},\"optionalNamespaces\":{\"eip155\":{\"chains\":[\"eip155:43114\",\"eip155:100\",\"eip155:42161\",\"eip155:137\",\"eip155:56\",\"eip155:250\",\"eip155:324\",\"eip155:10\",\"eip155:9001\",\"eip155:4689\",\"eip155:1088\"],\"methods\":[\"eth_sendTransaction\",\"personal_sign\",\"eth_accounts\",\"eth_requestAccounts\",\"eth_sendRawTransaction\",\"eth_sign\",\"eth_signTransaction\",\"eth_signTypedData\",\"eth_signTypedData_v3\",\"eth_signTypedData_v4\",\"wallet_switchEthereumChain\",\"wallet_addEthereumChain\",\"wallet_getPermissions\",\"wallet_requestPermissions\",\"wallet_registerOnboarding\",\"wallet_watchAsset\",\"wallet_scanQRCode\"],\"events\":[\"chainChanged\",\"accountsChanged\",\"message\",\"disconnect\",\"connect\"],\"rpcMap\":{\"1\":\"https://cloudflare-eth.com\",\"10\":\"https://mainnet.optimism.io\",\"56\":\"

**Comment by crypblizz8** [comment]
Hey there, what did the message reply from our explorer team say? 

The context you provided is quite limited. Thanks

**Reply by briannichols0702** [reply]
Hey I’ll send image of it
![IMG_1014](https://github.com/WalletConnect/walletconnect-monorepo/assets/125631412/a336a5f5-1e1c-4988-9978-4d74802fabea)

**Reply by briannichols0702** [reply]
Hope that helps some I can send u all test info from your environments if need be it tested good connected sign message no problem and sign typed data then also did the dapp test as well which worked fine as well and I hit the ping button thinking that pings u the results?

**Reply by briannichols0702** [reply]
Once again declined for same thing as I’m literally connected to the test environment 😂 you sure that test environment works correctly to show we tested on your end ???

**Reply by glitch-txs** [reply]
hey, you should test sign on the given website :) We track this specific dapp to make sure your wallet works properly :)

**Comment by briannichols0702** [comment]
We have done it 30 times all works then u decline

**Reply by glitch-txs** [reply]
could you share your project ID please?

**Comment by briannichols0702** [comment]
Project ID is
4c587effe43102265e0fd8fcd49b79d0 if you can send me a telegram DM
@safudev0702 I’ll send you screen recording of me connecting to your test
dapp and signing

**Reply by briannichols0702** [reply]
Uploading RPReplay_Final1693919153.mp4…
